CoreScriptsRoot/CoreScripts/Topbar.lua

```diff
@@ -1228,7 +1219,13 @@ if nameAndHealthMenuItem and topbarEnabled and not isTenFootInterface then
 	AddItemInOrder(RightMenubar, nameAndHealthMenuItem, RIGHT_ITEM_ORDER)
 end
 
-
+local Panel3D = require(GuiRoot.Modules.Panel3D)
+local function MoveHamburgerTo3D()
```

Comment:
Instead of moving the current Hamburger, we should disable the topbar hamburger and attach a 3d Hamburger to the panel.